### PR TITLE
added tasks resolution

### PR DESCRIPTION
**Pull request by NikaBuravska**

### CODE REVIEW

File: src/homework6/Calculator.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package homework6;
```
Comment (AndrewSkoba) [inline src/homework6/Calculator.java]: Pls, group your classes into packages more logically

File: src/homework6/Calculator.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+public class Calculator {
+    public static double add(double a,double b){
+        return a+b;
+    }
+    public static double subtract(double a,double b){
```
Comment (AndrewSkoba) [inline src/homework6/Calculator.java]: empty row between methods, Pls

File: src/homework6/Calculator.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
+package homework6;
+
+public class Calculator {
+    public static double add(double a,double b){
```
Comment (AndrewSkoba) [inline src/homework6/Calculator.java]: one space between methods params

File: src/homework6/Calculator.java [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+        return a+b;
+    }
+    public static double subtract(double a,double b){
+        return a-b;
+    }
+    public static double divide(double a,double b){
+        return a/b;
```
Comment (AndrewSkoba) [inline src/homework6/Calculator.java]: space between operators as well

File: src/homework6/Calculator.java (hunk 2 above)
Comment (AndrewSkoba) [inline src/homework6/Calculator.java]: space between brackets where params specified and body brackets

File: src/homework6/CalculatorValidation.java [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+        double sum = Calculator.add(number1, number2);
+        System.out.println(sum);
+
+        double devide = Calculator.divide(number1,number2);
+        System.out.println(devide);
+
+        double subtract = Calculator.subtract(number1,number2);
+        System.out.println(subtract);
+
+        double multiply = Calculator.multiply(number1,number2);
```
Comment (AndrewSkoba) [inline src/homework6/CalculatorValidation.java]: here you can use static import

File: src/homework6/User.java [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+        this.userName = userName;
+        this.age = age;
+        this.email = email;
+        setPassword(password);
+        this.isActive = isActive;
+        this.amountSpentMoney = amountSpentMoney;
+    }
+
+    public void setAge(int age) {
+       this.age=age;
```
Comment (AndrewSkoba) [inline src/homework6/User.java]: aligne, Pls

File: src/homework6/User.java [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+    }
+
+    public void setPassword(String password) {
+       this.password=password;
```
Comment (AndrewSkoba) [inline src/homework6/User.java]: same

File: src/homework6/User.java [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+    }
+
+    public int getAge() {
+        if(age<18){
```
Comment (AndrewSkoba) [inline src/homework6/User.java]: aligne as well

File: src/homework6/User.java [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+            System.out.println("Incorrect age");
+        }else {
+            this.age = age;
+        }
+        return age;
+    }
+
+    public String getPassword() {
+        if (password.length() <= 9) {
+            this.password = password;
```
Comment (AndrewSkoba) [inline src/homework6/User.java]: set var value it is not getter responsibility

File: src/homework6/User.java (hunk 8 above)
Comment (AndrewSkoba) [reply src/homework6/User.java]: set var value it is not getter responsibility

File: src/homework6/UsersValidation.java [hunk 10]
```diff
@@ -0,0 +1,30 @@
+package homework6;
+
+
+public class UsersValidation {
+    public static void main(String[] args) {
+        User user1 = new User("Ola",25,"test@gmail.com","test53",true,234.5);
```
Comment (AndrewSkoba) [inline src/homework6/UsersValidation.java]: spaces between ','

File: src/homework6/Calculator.java (hunk 1 above)
Comment (NikaBuravska) [reply src/homework6/Calculator.java]: done

File: src/homework6/Calculator.java (hunk 2 above)
Comment (NikaBuravska) [reply src/homework6/Calculator.java]: done

File: src/homework6/Calculator.java (hunk 3 above)
Comment (NikaBuravska) [reply src/homework6/Calculator.java]: done

File: src/homework6/Calculator.java (hunk 2 above)
Comment (NikaBuravska) [reply src/homework6/Calculator.java]: done

File: src/homework6/Calculator.java (hunk 4 above)
Comment (NikaBuravska) [reply src/homework6/Calculator.java]: done

File: src/homework6/User.java (hunk 6 above)
Comment (NikaBuravska) [reply src/homework6/User.java]: done

File: src/homework6/User.java (hunk 7 above)
Comment (NikaBuravska) [reply src/homework6/User.java]: done

File: src/homework6/User.java (hunk 8 above)
Comment (NikaBuravska) [reply src/homework6/User.java]: done

File: src/homework6/User.java (hunk 9 above)
Comment (NikaBuravska) [reply src/homework6/User.java]: done

File: src/homework6/UsersValidation.java (hunk 10 above)
Comment (NikaBuravska) [reply src/homework6/UsersValidation.java]: done

**Review by AndrewSkoba — APPROVED** [review]

**Pull request merged**